src/notification-service.ts

```diff
@@ -0,0 +1,39 @@
+import * as _ from 'lodash';
+const webpush = require('web-push');
```

Comment:
Should we import this as ES import as well?
`import * as webpush from 'web-push';`

Reply:
noch besser... es gibt typings: e7d4781c1eebf3b5ab74688115474010ff7f3555